lib/sidekiq/job_util.rb

```diff
@@ -94,16 +94,16 @@ def json_safe?(item)
     end
 
     def json_unsafe_item(item)
-      case item
-      when String, Integer, Float, TrueClass, FalseClass, NilClass
+      klass = item.class
+      if [String, Integer, Float, TrueClass, FalseClass, NilClass].include?(klass)
```

Comment:
Promote this Array to a Constant otherwise it will allocate a new array for every call.

Reply:
I can do that, but this method will be called only on exceptional cases when we have an unsafe json. Is it worth it?

Reply:
Oh I see, I forgot RECURSIVE_JSON_SAFE is separate. Can we combine the two codepaths? Can RECURSIVE_JSON_SAFE change to return the unsafe item, so `json_safe?` becomes `json_unsafe?`?